server/app/assets/javascripts/session.ts

```diff
@@ -123,39 +160,48 @@ export class SessionTimeoutHandler {
           this.showWarning(WarningType.INACTIVITY)
           this.hasInactivityWarningBeenShown = true
         },
+        type: WarningType.INACTIVITY,
```

Comment:
question (non-blocking): Where are we using `timeout.type`? I'm not seeing it.

Reply:
I used it for logging and using it for unit tests but since then I have changed the tests and removed logs from session.ts so removed the timeout.type.